Mathlib/MeasureTheory/Integral/SetIntegral.lean

```diff
@@ -403,35 +403,23 @@ theorem set_integral_eq_integral_of_forall_compl_eq_zero (h : ∀ x, x ∉ s →
   set_integral_eq_integral_of_ae_compl_eq_zero (eventually_of_forall h)
 #align measure_theory.set_integral_eq_integral_of_forall_compl_eq_zero MeasureTheory.set_integral_eq_integral_of_forall_compl_eq_zero
 
+-- TODO: Do we need these 2 lemmas?
```

Comment:
@kkytola You added these lemmas. Do we need them for something other than `set_integral_eq_integral_of_ae_restrict_eq_zero`?

Reply:
I believe these are not needed for anything else, and then transitively they are not needed at all if that lemma is removed.